vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditSpaceLinksController.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+public class EditSpaceLinksController {
+
+    public ResponseEntity<String> checkIfSpaceExists(ISpaceManager spaceManager, String id, String x, String y) throws IOException{
```

Comment:
these methods should not be public unless they are indented to be used outside of the inheriting classes (which they are not, are they?).